core/src/Exceptions/CoreErrorMessages.cs

```diff
@@ -51,7 +51,8 @@ internal class CoreErrorMessages
         public const string UnsupportedAuthorityValidation =
             "Authority validation is not supported for this type of authority";
 
-        public const string AuthenticationCanceled = "User canceled authentication";
+        public const string AuthenticationCanceled = "User canceled authentication. On an Android device, this could be due to the system browser." +
```

Comment:
>due to the system browser. [](start = 120, length = 26)

suggestion: due to the lack of capabilities such as custom tabs for the system browser.